moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -1307,7 +1314,7 @@ bool IKFastKinematicsPlugin::getPositionIK(const std::vector<geometry_msgs::Pose
   }
   else
   {
-    ROS_DEBUG_STREAM_NAMED("ikfast", "No IK solution");
+    ROS_DEBUG_STREAM_NAMED(name_, "No IK solution");
```

Comment:
I am really happy to see this cleanup change but for future reference this would be ideally in a separate CL than substantial functionality change

Reply:
@davetcoleman my bad. I asked for that change and should have mentioned that it belongs in it's own PR.

Reply:
Thank you for your reviews. I will keep that in mind for the future.